server/settings.py

```diff
@@ -143,7 +143,10 @@ def env(variable, fallback_value=None):
     'apps.packages',
     'apps.privilege',
     'apps.rule_sets',
-    'apps.highlights'
+    'apps.highlights',
+    'apps.macros',
+
+    'macros.currency',
```

Comment:
I think it needs to be dynamic. So I guess settings.py doesn't need to know about the individual modules.